client/webserver/site/src/js/doc.ts

```diff
@@ -20,6 +20,7 @@ const BipIDs: Record<number, string> = {
   3: 'doge',
   145: 'bch',
   60: 'eth',
+  136: 'firo',
```

Comment:
Please resize the firo.png to 100x100 and run it through optipng, and firo.svg through svgo.

Reply:
As requested .. fixed in https://github.com/decred/dcrdex/pull/2270/commits/e111e14029f7062e554ebb3fe0e59490bd98e8d7